app/build.gradle

```diff
@@ -3,6 +3,7 @@ plugins {
     id 'org.jetbrains.kotlin.android'
     id 'kotlin-kapt'
     id 'com.google.dagger.hilt.android'
+    id 'com.google.android.libraries.mapsplatform.secrets-gradle-plugin'
```

Comment:
What's the use of this plugin?

Reply:
It helps import properties from local.properties in project. That is where I have kept my api key.